src/rdkafka_topic.c

```diff
@@ -189,6 +189,27 @@ rd_kafka_topic_t *rd_kafka_topic_find0_fl(const char *func,
         return rkt;
 }
 
+/**
```

Comment:
As in previous function documentation we can avoid duplicating the whole comment by writing:
```c
/**
 * Same semantics as ..find() but takes a Uuid instead.
 */
```